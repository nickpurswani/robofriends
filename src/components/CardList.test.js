import React from 'react';
import { render } from '@testing-library/react';
import CardList from './CardList';

const filteredRobots = [{
  id: 1,
  name: 'Leanne Graham',
  username: 'Bret',
  email: 'Sincere@april.biz'
}];

test('renders without crashing', () => {
  const { asFragment } = render(<CardList robots={filteredRobots} />);
  expect(asFragment()).toMatchSnapshot();
});
