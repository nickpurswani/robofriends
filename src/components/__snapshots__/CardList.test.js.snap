// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div>
    <div
      class=" tc bg-light-green dib br4 pa3 ma2 grow bw2 shadow-5"
    >
      <img
        alt="robots"
        src="https://robohash.org/1?200*200"
      />
      <div>
        <h2>
          Leanne Graham
        </h2>
        <p>
          Sincere@april.biz
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
